### ISSUE TITLE
[BUGFIX] Can not make ImageReference in ThumbnailBox

### ISSUE THREAD

**Issue by fabarea** [bootstraptheme-for-typo3/fluidbootstraptheme#289]
I am experimenting a PHP fatal when trying to create an ImageReference. The PHP error is to be seen in the Ajax request.

```
#1: PHP Catchable Fatal Error: Argument 1 passed to TYPO3\CMS\Backend\Controller\FormInlineAjaxController::addFlexFormDataStructurePointersFromAjaxContext() must be of the type array, null given, called in 
/Users/fudriot/Sites/Typo3Cms/idebug.lan/vendor/typo3/cms/typo3/sysext/backend/Classes/Controller/FormInlineAjaxController.php on line 74 and defined in 
vendor/typo3/cms/typo3/sysext/backend/Classes/Controller/FormInlineAjaxController.php line 873 (More information)
```

Tested with `flux` and `fluidbootstraptheme` development branch on TYPO3 7 LTS.

![screenshot 2015-12-15 16 15 36](https://cloud.githubusercontent.com/assets/620730/11814523/7823f0ec-a347-11e5-8e42-a918ee7d185f.png)

**Comment by Terminator81** [comment]
Hello, do you have a solution for this problem?

**Comment by outdoorsman** [comment]
I am also getting this error when I try to "Create a new relation" to an image in the Image Gallery content element. I'm using TYPO3 7.6.4. Here's some of the errors I'm seeing...

## A fluidbootstraptheme error

`Core: Exception handler (WEB): Uncaught TYPO3 Exception: #1: PHP Warning: filesize(): stat failed for /home/example/public_html/typo3temp/Cache/Code/fluid_template/Fluidbootstraptheme_Standard_action_index_blahblahblahhexnumbers.php in /home/example/public_html/vendor/typo3/cms/typo3/sysext/core/Classes/Cache/Backend/FileBackend.php line 331 | TYPO3\CMS\Core\Error\Exception thrown in file /home/example/public_html/vendor/typo3/cms/typo3/sysext/core/Classes/Error/ErrorHandler.php in line 111. Requested URL: http://www.example.com/typo3/index.php?M=web_layout&moduleToken=blahblahblahhexnumbers`

## Another error that I think is the 500 error that's happening when I try to make image references

`Core: Exception handler (WEB): Uncaught TYPO3 Exception: #1446996319: Configuration retrieved from FlexForm is incomplete or not of type "inline". | UnexpectedValueException thrown in file /home/example/public_html/vendor/typo3/cms/typo3/sysext/backend/Classes/Controller/FormInlineAjaxController.php in line 935. Requested URL: http://www.example.com/typo3/index.php?ajaxID=%2Fajax%2Frecord%2Finline%2Fcreate&ajaxToken=blahblahblahhexnumbers`

**Comment by outdoorsman** [comment]
After #292 it seems that the error doesn't happen any more. On of the things that was changed was that the `flux`, `fluidcontent`, `fluidpages`, and `vhs` have `require-dev` set to `dev-development` now, which I believe is what fixed the issue at least for TYPO3 7.6.4.
